docs/team/junwei3112.md

```diff
@@ -10,8 +10,46 @@ The user interacts with it using a CLI, and it has a GUI created with JavaFX. It
 
 Given below are my contributions to the project:
 
-* **Documentation**:
-  * User Guide:
-    * Added documentation for the feature `list`
-  * Developer Guide:
+* **Code contributed**:
+  * My contributions to the code base can be found [here](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=W16-3&sort=groupTitle&sortWithin=title&since=2021-09-17&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabAuthor=JunWei3112&tabRepo=AY2122S1-CS2103T-W16-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other&authorshipIsBinaryFileTypeChecked=false&tabType=authorship).
+
+* **Enhancements implemented**:
+  * Implemented the functionality of the `paid` and `unpaid` commands for v1.2.
+    * Relevant PRs: [#63](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/63)
+    * Allows private tutors to set and unset payment status of their students.
+    * Removed from TutorAid for v1.4 due to its limited functionality.
+  * Set up the storage system for TutorAid to store and fetch `Lesson` objects from memory.
+    * Relevant PRs: [#90](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/90).
+    * Worked closely with the `Storage` and `Model` components to implement storing, retrieving and handling of `Lesson` objects within TutorAid.
+    * It was quite hard to implement since there were no concrete methods implemented to modify and view lessons in TutorAid at the time. As such, I wrote extensive test cases, alongside my functional code, to make sure that the storage system worked properly.
+  * Implemented the functionality of the `add -l` and `del -l` commands.
+    * Relevant PRs: [#119](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/119).
+    * Allows private tutors to add and delete lessons with the specified lesson details.
+
+* **Contributions to the UG**:
+  * Added documentation for the following features in the `Features` and `Command summary` sections
+    * `list`
+    * `add -l`, `del -l`
+
+* **Contributions to the DG**:
+  * Updated the documentation and class diagram for the `Storage` component under `Architecture`.
+  * Added documentation and UML diagrams for the implementation details of `add -l`.
+  * Contributed to the following sections:
     * Target user profile, value proposition, user stories, use cases, non-functional requirements, glossary
+
+* **Contributions to team-based tasks**:
+  * Set up the GitHub team organisation and repository.
+  * Set up Codecov.
+  * In charge of maintaining the issue tracker and milestones.
+  * Update `index.md` to fit TutorAid.
+
+* **Review/mentoring contributions**:
```

Comment:
Maybe you can include just some of the PRs to which you value-added (and even include a short sentence on how you had contributed to the PR) so that it is easier for readers to understand? 

(Just in case might be good to check with someone on whether you need to link all of your PRs here before making any of the above suggested changes)

Reply:
The PPP that I referenced just linked a few review comments to highlight (non-trivial examples / helpful comments)

Reply:
Yes I have changed it to include some of the more value-adding PRs. Thanks to both!